subscribie/blueprints/admin/templates/admin/settings/rename_shop.html

```diff
@@ -0,0 +1,47 @@
+{% extends "admin/layout.html" %}
+{% block title %} Rename Shop Address{% endblock %}
+
+{% block body %}
+
+<h2 class="text-center text-dark mb-3">Rename Shop Address</h2>
+
+<div class="container">
+  <ul class="breadcrumb">
+    <li class="breadcrumb-item"><a href="/">Shop</a></li>
+    <li class="breadcrumb-item"><a href="{{ url_for('admin.dashboard') }}">Manage My Shop</a></li>   
+    <li class="breadcrumb-item active" aria-current="page">Rename Shop Address</li>
+  </ul>
+</div>
+
+<main>
+  <div class="section">
+    <div class="container">
+
+      <div class="row row-cols-1 row-cols-md-2">
+        <div class="col-md-7">
+          <h2>Change Your Shop Web Address</h2>
+          <p>The URL of your shop will be changed as long as the name is available
+              and the name contain only letter and numbers</p>
+	      <p>Please Write your New web address without the .subscriby.shop
```

Comment:
@joeltejeda 

Please:
- remove line 25
- Because we can/should do handle this server side in  subscribie/blueprints/admin/__init__.py for the user automatically, in the rare case the user may enter the full address

e.g. 

```
"my-new-name.subscriby.shop".replace("subscriby.shop","").replace(".","")
```
Regardless the alnum check would disallow it :+1: